src/Orleans.Core.Abstractions/CodeGeneration/ActivationAttribute.cs

```diff
@@ -72,4 +72,10 @@ public SerializerAttribute(Type targetType)
         {
         }
     }
+
+    /// <summary>Identifies generated code.</summary>
+    [AttributeUsage(AttributeTargets.Class)]
+    public sealed class OrleansGeneratedCodeAttribute : Attribute
```

Comment:
Can't we just leverage [System.CodeDom.Compiler.GeneratedCodeAttribute](https://msdn.microsoft.com/en-us/library/system.codedom.compiler.generatedcodeattribute(v=vs.110).aspx) ?

Reply:
I think we should stop leveraging that, since it's common for all code generators, but I can change it.

Reply:
But are we actually depending on this attribute or are we adding it for other tools? (like static analyzers, which skip code that has this attribute).
Also the attribute in CodeDom takes the tool name, so we can pass "Orleans" there to differentiate.